.github/workflows/release.yaml

```diff
@@ -21,3 +21,17 @@ jobs:
         id: release
         with:
           token: ${{ steps.generate_token.outputs.token }}
+      - uses: actions/checkout@v4
+      - name: tag major and minor versions
+        run: |
+          git config user.name czi-github-helper[bot]
+          git config user.email 95879977+czi-github-helper[bot]@users.noreply.github.com
+          git tag -d v${{ steps.release.outputs.major }} || true
+          git tag -d v${{ steps.release.outputs.major }}.${{ steps.release.outputs.minor }} || true
+          git push origin :v${{ steps.release.outputs.major }} || true
+          git push origin :v${{ steps.release.outputs.major }}.${{ steps.release.outputs.minor }} || true
+          git tag -a v${{ steps.release.outputs.major }} -m "Release v${{ steps.release.outputs.major }}"
+          git tag -a v${{ steps.release.outputs.major }}.${{ steps.release.outputs.minor }} -m "Release v${{ steps.release.outputs.major }}.${{ steps.release.outputs.minor }}"
+          git push origin v${{ steps.release.outputs.major }}
+          git push origin v${{ steps.release.outputs.major }}.${{ steps.release.outputs.minor }}
+        if: ${{ steps.release.outputs.release_created }}
```

Comment:
This is great, what do you think about putting it in a composite action (maybe with the steps above it or maybe not, I'm not really sure)? I ask because I know @jakeyheath wants to do something similar for our Terraform modules

Reply:
I like this idea. the TF PR is at https://github.com/chanzuckerberg/shared-infra/pull/10726 and I think you're right it makes sense to consolidate

Reply:
this is going to be great, I agree having with having a consolidated action

Reply:
done.

instead of spending time getting massaging this to be testable I'm thinking I merge this in and quickly fix any issues that come up.  sounds good?

Reply:
I wouldn't have it any other way